wp-content/themes/core/assets/css/src/theme/typography/_variables.pcss

```diff
@@ -6,43 +6,25 @@
 
 :root {
 	/* -----------------------------------------------------------------------------
-	 * Font Stacks
+	 * Font Families
 	 * ----------------------------------------------------------------------------- */
 
 	--font-family-core-icons: 'core-icons';
-	--font-family-proxima-nova: "proxima-nova", "Helvetica Neue", Helvetica, Arial, sans-serif;
-	--font-family-sans-serif: "Helvetica Neue", Helvetica, Arial, sans-serif;
-	--font-family-base: var(--font-family-proxima-nova);
+	--font-family-serif: serif;
+	--font-family-sans-serif: sans-serif;
+	--font-family-base: var(--font-family-sans-serif);
 
 	/* -----------------------------------------------------------------------------
 	 * Font Sizing
 	 * ----------------------------------------------------------------------------- */
 
-	--font-size-base: 18px;
-	--font-size-content: 18px;
-	--font-size-content-s: 16px;
-
-	/* -----------------------------------------------------------------------------
-	 * Font Weights
-	 * ----------------------------------------------------------------------------- */
-
-	--font-weight-light: 300;
-	--font-weight-regular: 400;
-	--font-weight-semibold: 600;
-	--font-weight-bold: 700;
-
-	/* -----------------------------------------------------------------------------
-	 * Line Heights
-	 * ----------------------------------------------------------------------------- */
-
-	--line-height-base: 1.5;
-	--line-height-m: 1.33;
-	--line-height-s: 1.25;
-	--line-height-ss: 1.15;
-	--line-height-even: 1;
+	--font-size-body-large: 20px;
+	--font-size-body: 16px;
+	--font-size-body-small: 12px;
 
 	/* -----------------------------------------------------------------------------
 	 * Border Radius
+	 * TODO: Where do these belong?
```

Comment:
Good question, have the same q for like border colors too

Reply:
Yup. Thinking that this will maybe end up landing in a  theme/decorations directory eventually. I see some box-shadows stuff in the designs as well now.